pyomo/contrib/preprocessing/plugins/constraint_tightener.py

```diff
@@ -19,9 +20,16 @@ class TightenContraintFromVars(IsomorphicTransformation):
     For now, this only operates on linear constraints.
 
     """
+    class _MissingArg(object):
+        pass
+
+    def _apply_to(self, model, rounding_ndigits=_MissingArg, **kwds):
```

Comment:
Why are you adding `*kwds` to the signature?  It is not used, nor is it verified to be empty.

Reply:
It is not used, not. No reason other than conforming to the same pattern as many other `TransformationFactory`s (eg. `BigM_Transformation`). Happy to revert if you'd prefer that?